Source/Core/Core/IOS/Network/KD/NetKDRequest.h

```diff
@@ -17,6 +18,7 @@
 #include "Core/IOS/Network/KD/Mail/WC24Send.h"
 #include "Core/IOS/Network/KD/NWC24Config.h"
 #include "Core/IOS/Network/KD/NWC24DL.h"
+#include "Core/IOS/Network/KD/NetKDTime.h"
```

Comment:
It seems clang-format doesn't sort it alphabetically correctly _(or rather its sorting is not case insensitive, probably ASCII based since lowercase is after uppercase)_. Feel free to ignore this comment since there is not much you can do as it isn't related to this PR.